### DISCUSSION TITLE
Transcript randomly switches to all caps

### DISCUSSION THREAD

**Discussion by michaelcantrell** [deepgram/community#291] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

We have started seeing random portions of the transcript coming through as all caps in the middle of people's speech.

`They were sent in let's see. Some two clerks SOME TO ALL OVER TO PLAY SOMS TO ME. Yeah, I'LL SENDALS TO YOU.`

I've seen this specifically in one file, but it may be happening in others. This was the only report from a customer about it. This was first noticed on Friday, July 21st. The transcription is still coming through that way today. The uppercase text is also significantly less accurate than the regular text.

We need to know why this is happening and what is being done to fix it.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?language=en-US&model=nova&smart_format=true&diarize=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

54b59c7c-9222-4d28-ac53-ae8dd62833f2

### If possible, please attach an example audio file to reproduce the issue.

I have an m4a that cannot be uploaded here. It can be provided on request.

**Comment by jjmaldonis** [comment]
Hi @michaelcantrell, this is a very uncommon situation. Have you seen this in more than one file? or is the customer rerunning the same file?

It looks like you've reached out to us via multiple channels - this GitHub discussion is a good spot to have the conversation and understand what's going on. In this particular instance, this is a very rare occurrence from what we have seen. We understand the underlying issue and are building new models that will fix it, along with other overall improvements to the transcript and accuracy. So you can be sure that this is something we are aware of and are taking into consideration as we improve transcriptions across the board.

**Reply by michaelcantrell** [reply]
We haven't seen it on more than one file yet, which is why we were so thrown off by it randomly appearing. This being the only reported issue to us doesn't mean it isn't happening on other files, just that we don't know about it.

When you say it is "very rare", how do you quantify that?

**Reply by jjmaldonis** [reply]
That's good to know this is the only file you've noticed it on. I don't have solid numbers for how often this occurs, but our users will often reach out to let us know about things like this. To my knowledge, we've gotten feedback about this a few times since Nova was released in April. Our future models should not have this issue.

**Reply by michaelcantrell** [reply]
@jjmaldonis Is there a rough timeline on when the new model might be released?

**Reply by jjmaldonis** [reply]
I cannot provide an ETA at this time.

**Comment by jpvajda** [comment, marked as answer]
@michaelcantrell  👋  I thought I'd share Nova-2 was just released GA today, you might want to give it a try for your use case to see if there are any improvements. https://deepgram.com/learn/nova-2-speech-to-text-api
